pkgs/development/compilers/llvm/5/llvm.nix

```diff
@@ -36,9 +36,7 @@ in stdenv.mkDerivation (rec {
   nativeBuildInputs = [ cmake python ]
     ++ stdenv.lib.optional enableManpages python.pkgs.sphinx;
 
-  buildInputs = [ libxml2 libffi ]
-    # TODO(@Ericson2314): Remove next mass rebuild
-    ++ stdenv.lib.optionals (stdenv.isDarwin && stdenv.hostPlatform == stdenv.buildPlatform) [ libcxxabi ];
+  buildInputs = [ libxml2 libffi ];
```

Comment:
Oh for the record this is because `compiler-rt` is now built separately and `libcxxabi` was just needed for that. Overall on the LLVM side, tools and runtime libraries are never built in the same derivation, which is good.